scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -216,20 +220,29 @@ def __init__(self, test: dict, config: dict, definitions: SpecDefinitions, pics_
             test, 'timedInteractionTimeoutMs')
         self.busy_wait_ms = _value_or_none(test, 'busyWaitMs')
         self.wait_for = _value_or_none(test, 'wait')
+        self.event_number = _value_or_none(test, 'eventNumber')
 
         self.is_attribute = self.command in _ATTRIBUTE_COMMANDS or self.wait_for in _ATTRIBUTE_COMMANDS
```

Comment:
The `is_attribute` and `is_event` are incorrect when you are looking for `waitForReport` for an event subscription